exporter/influxdbexporter/config.go

```diff
@@ -51,6 +51,18 @@ type Config struct {
 	// - https://github.com/open-telemetry/opentelemetry-collector/tree/main/semconv
 	SpanDimensions []string `mapstructure:"span_dimensions"`
 
+	// LogRecordDimensions are log record attributes to be used as line protocol tags.
+	// These are always included as tags, if available:
+	// - trace ID
+	// - span ID
+	// The default values:
+	// - service.name
+	// Other common attributes can be found here:
+	// - https://github.com/open-telemetry/opentelemetry-collector/tree/main/semconv
```

Comment:
I would say this link is easier to read
```suggestion
	// - https://opentelemetry.io/docs/specs/semconv/
```

Reply:
Agreed! I've also made this change a little earlier in the same file.